user_modules/shared_testcases/testCasesForPolicyAppIdManagament.lua

```diff
@@ -0,0 +1,77 @@
+---------------------------------------------------------------------------------------------
+-- Policy: AppID Management common module
+---------------------------------------------------------------------------------------------
+
+local common = {}
+
+local commonFunctions = require("user_modules/shared_testcases/commonFunctions")
+
+local policy_file_path = commonFunctions:read_parameter_from_ini("SystemFilesPath")
```

Comment:
@dtrunov Name of function is updated in f583c49bd8a5c1a3f81203eb9bf11a5336819a94
But it should stay uncommented. Please clarify with Igor Kovalenko this point